### DISCUSSION TITLE
Add a mod option to Strict Tracking to change the size of the follow circle

### DISCUSSION THREAD

**Discussion by ashandelle** [ppy/osu#30438] [Ideas]
Strict Tracking makes sliders harder by forcing the player to remain inside the follow circle but has no punishment for leaving the slider itself, I think this is fine as a default but that there should be a customization slider to change the size of follow circle, 1.0 for the same width as the slider, 2.0 (I cant really tell) for default size, and maybe an extended range like 0.5-3.0. Also #36665 and #38172.

**Comment by Skylandia** [comment]
I've been playing [Britney Spears - Toxic [Emilia's Expert]](https://osu.ppy.sh/beatmapsets/1061947#osu/2251926) and this option would be a perfect way to increase the difficulty.

**Reply by ashandelle** [reply]
If you like that I have a very large collection of maps that I play ST with: https://osu.ppy.sh/multiplayer/rooms/1509632, its slightly outdated though (800+ now)
